packages/shared/src/lib/auxiliary/wallet-connect/stores/persisted-dapp-namespaces.store.ts

```diff
@@ -5,25 +5,25 @@ import { getActiveProfile } from '@core/profile/stores'
 
 interface IPersistedNamespaces {
     [profileId: string]: {
-        [dappId: string]: SupportedNamespaces
+        [dappOrigin: string]: SupportedNamespaces
     }
 }
 
 export const persistedDappNamespaces: Writable<IPersistedNamespaces> = persistent('persistedDappNamespaces', {})
 
-export function getPersistedDappNamespacesForDapp(dappId: string): SupportedNamespaces | undefined {
+export function getPersistedDappNamespacesForDapp(dappOrigin: string): SupportedNamespaces | undefined {
```

Comment:
Suggestion: maybe make it `dappUrl` or `dappOriginUrl` as this better reflects what it is?